packages/flutter/lib/src/material/input_border.dart

```diff
@@ -116,6 +116,99 @@ class _NoInputBorder extends InputBorder {
   }
 }
 
+/// Creates an [InputBorder] that has no painted outline and rounded corners.
+///
+/// The input decorator's "container" is the optionally filled area above the
```

Comment:
This statement seems to come out of nowhere.

Reply:
Removed it.